tests/unit/test_workflow_files.py

```diff
@@ -2050,3 +2071,31 @@ def test_delete_runN_skipif_cleanedrun_not_runN(tmp_path):
     (tmp_path / 'runN').symlink_to(tmp_path / 'run2')
     clean(str(tmp_path.name) + '/' + 'run1', tmp_path / 'run1')
     assert sorted([i.stem for i in tmp_path.glob('*')]) == ['run2', 'runN']
+
+
+@pytest.mark.parametrize(
+    'workflow_name, err_expected',
+    [
+        ('foo/' * (MAX_SCAN_DEPTH - 1), False),
+        ('foo/' * MAX_SCAN_DEPTH, True)  # /run1 takes it beyond max depth
+    ]
+)
+def test_install_workflow__max_depth(
+    workflow_name: str,
+    err_expected: bool,
+    tmp_path: Path,
+    tmp_run_dir: Callable,
```

Comment:
We already have a `test_dir` / `run_dir` fixture.

Reply:
Hmm but those are in `tests/integration/conftest` whereas this test is in `tests/unit/`

Reply:
Ah, this is more an integration test than a unit test since it's testing a full CLI function involving FS.

Can leave it here, we have bigger fish to fry right now, but probably best to move those fixtures into the test module to prevent them being re-used.